sdk/communication/arm-communication/README.md

```diff
@@ -63,8 +63,47 @@ const client = new CommunicationServiceManagementClient(new DefaultAzureCredenti
 // });
 // const client = new CommunicationServiceManagementClient(credential, subscriptionId);
 ```
+### Create Communication Service Resource with 'Managed Identity'
+
+This example creates a new Azure Communication Service resource with a system assigned managed identity. The [beginCreateOrUpdate] method is used to create the resource and assign the managed identity. 
+
+System Assigned Identity is an identity created and managed by Azure. When you enable a system assigned managed identity, an identity is created in Azure AD, which is tied to the lifecycle of that service instance. A system-assigned identity is tied to your application and is deleted if your app is deleted. An app can only have one system-assigned identity. 
+
+```javascript
+// Existing resource creation code
+client.communicationServices.beginCreateOrUpdate(resourceGroup, resourceName, resource)
+
+// Create Resource with Managed Identity
+const resource = {
+    location: "Global",
+    dataLocation: "United States",
+    identity: {
+        type: "SystemAssigned" 
+    }
+};
+
+// Defining an object that specifies identity type as System Assigned for the new resource.
```

Comment:
The next line of code doesn't define an object, it creates a resource. Kindly correct comment.